tests/templates/kuttl/smoke/10-deploy-spark-app.yaml.j2

```diff
@@ -26,7 +26,11 @@ spec:
   driver:
     logging:
       enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+    jvmSecurity:
+      networkaddress.cache.ttl: "5"
```

Comment:
I personally like to have the smoke tests as simply as possible to be really smoke. This way we guarantee the defaults are sufficient to have a running setup and no parameters need to be tweaked (e.g. decrease the cache ttl) to have a running setup)

Reply:
Done https://github.com/stackabletech/spark-k8s-operator/pull/272/commits/6106b0848d008264d74bbac23bdc27ed2de95eab